### PR TITLE
Updates made to match changes in Watchmaker

### LINKED ISSUE #6: Cloudformation templates doesn't allow config.yaml from other sources except `http[s]` url
Presently, all the Cloudformation templates only allow `http[s]` paths to the Watchmaker`config.yaml` file.  Watchmaker has been updated to accept sources from AWS S3 directly and local folders.

### PR DESCRIPTION
**Pull request by eemperor**
The changes in this pull requests include updates made to be in sync with Watchmaker.

Specific items addressed:
- Fixes #6 
- Updates Window Python installer url to v3.6.4.

### CODE REVIEW

File: modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json [hunk 1]
```diff
@@ -519,9 +506,8 @@
             "Type": "String"
         },
         "WatchmakerConfig": {
-            "AllowedPattern": "^$|^http[s]?://.*$",
```
Comment (lorengordon) [inline modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json]: I'd like to keep the `AllowedPattern`. It's a regex, so we should be able to include several options.

Maybe this: `"^$|^(http[s]?|file|s3)://.*$"`. That would not allow paths like `/` or `C:`, but those can be passed as `file://...` instead.

https://regex101.com/r/aSi2LD/1

Comment (eemperor) [reply modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json]: I agree.  I'll make the recommended changes.

Comment (eemperor) [reply modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json]: Just realized you may be referring to only to the Linux templates?  I made the changes to the Windows ones as well.  I did test with "file://%TEP%/config.yml" on a Windows instance and it still worked properly.

Comment (lorengordon) [reply modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json]: All templates. The new uri methods support `file://` on windows as well. Examples in the pr that merged the new approach: https://github.com/plus3it/watchmaker/pull/507

**Pull request merged**